lookout/style/typos/analyzer.py

```diff
@@ -69,8 +70,13 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
         base_files_by_lang = files_by_language(c.base for c in changes)
         head_files_by_lang = files_by_language(c.head for c in changes)
         line_length = self.config.get("line_length_limit", self.DEFAULT_LINE_LENGTH_LIMIT)
+        overall_size = self.config.get("overall_size_limit", self.DEFAULT_OVERALL_SIZE_LIMIT)
```

Comment:
I think it is worth to have the same structure or config loading as we have in `FormatAnalyser`.
But I see that you just keep the same stricture, so we agree on this we can leave it for @irinakhismatullina :)